src/main/java/core/selenium/webdrivers/Chrome.java

```diff
@@ -5,7 +5,19 @@
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
 
+/**
+ * Firefox, this class is in charge of initializing the specific characteristics of Chrome.
```

Comment:
Please change the class name in the comment.

Reply:
Solved.